i6_models/parts/e_branchformer/merge.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+__all__ = ["MergerV1Config", "MergerV1"]
+
+from dataclasses import dataclass
+
+import torch
+from torch import nn
+
+from i6_models.config import ModelConfiguration
+
+
+@dataclass
+class MergerV1Config(ModelConfiguration):
+    """
+    Attributes:
+        input_dim: input dimension
+        kernel_size: kernel size of the depthwise convolution layer
+        dropout: dropout probability
+    """
+
+    input_dim: int
+    kernel_size: int
```

Comment:
Are we allowed to have even kernel_size here?

Reply:
I think even `kernel_size` is not forbidden, but usually an odd kernel_size is used for the convolution so I follow the valid check as for the other convolution in i6_model for standardization. Do you suggest that we remove the check here?